htlcswitch/switch.go

```diff
@@ -589,7 +599,10 @@ func (s *Switch) ForwardPackets(packets ...*htlcPacket) chan error {
 	// left in a half added state, which can happen when recovering from
 	// failures.
 	for _, packet := range failedPackets {
-		failure := lnwire.NewTemporaryChannelFailure(nil)
+		update, _ := s.cfg.FetchLastChannelUpdate(
```

Comment:
We should only have to fetch this once, since all adds will be coming from the same incoming link